src/test/java/lotto/domain/LotteryRankTest.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+class LotteryRankTest {
+    @Test
+    void LotteryRank_유효한등수계산() {
+        assertThat(LotteryRank.toLotteryRank(6, true).rank).isEqualTo(1);
```

Comment:
`LotteryRank.rank`가 꼭 있어야 할까요?
```suggestion
        assertThat(LotteryRank.toLotteryRank(6, true)).isEqualTo(LotteryRank.RANK1);
```